src/Symfony/Bundle/SecurityBundle/composer.json

```diff
@@ -28,7 +28,7 @@
         "symfony/password-hasher": "^6.4|^7.0",
         "symfony/security-core": "^6.4|^7.0",
         "symfony/security-csrf": "^6.4|^7.0",
-        "symfony/security-http": "^6.4|^7.0",
+        "symfony/security-http": "^7.1",
```

Comment:
do we really need this? I'm not sure

Reply:
If this version constraint is not updated to 7.1, we have to detect the version in order to create the service alias or not.

Reply:
Maybe I'm wrong, since the interface don't have to exist for it's name to be used as a service name.

Reply:
why? the alias is just a string, we don't need the interface to be here to have it work

Reply:
Reverted the constraint changes.